DQMOffline/Trigger/plugins/JetDQM.h

```diff
@@ -0,0 +1,63 @@
+#ifndef DQMOffline_Trigger_JetDQM_H
+#define DQMOffline_Trigger_JetDQM_H
+
+#include <vector>
+
+#include "DQMOffline/Trigger/plugins/GENERICDQM.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "DataFormats/JetReco/interface/PFJet.h"
+#include "DataFormats/METReco/interface/PFMET.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+class JetDQM : public GENERICDQM
+{
+ public:
+  JetDQM();
+  virtual ~JetDQM();
+
+  void initialise(const edm::ParameterSet& iConfig);
+  void bookHistograms(DQMStore::IBooker &);
+  void fillHistograms(const std::vector<reco::PFJet> & jets,
+		      const reco::PFMET & pfmet,
+		      const int & ls,
+		      const bool passCond);
+  static void fillJetDescription(edm::ParameterSetDescription & histoPSet);
+
+  //keep public to fill them directly....
+  std::vector<double> jetpt_variable_binning_;
```

Comment:
please move these to be private until some code needs them to be public (not sure I know what sort of code that would be?)